readme.txt

```diff
@@ -108,6 +108,16 @@ Web Stories are powered by [AMP](https://amp.dev/), which adds some restrictions
 
 For the plugin's full changelog, please see [the Releases page on GitHub](https://github.com/google/web-stories-wp/releases).
 
+= 1.6.0 =
+
+**Release Date:** April 13, 2021.
+
+* New feature: automated video optimization during upload.
```

Comment:
Should we mention support for .mov files?

Reply:
```suggestion
* New feature: automated video optimization during upload, enabling support for MOV files.
```